mockld/roku.go

```diff
@@ -28,7 +28,7 @@ func (srv *RokuServer) Wrap(h http.Handler) http.Handler {
 			return
 		}
 
-		userJSONBase64 := base64.StdEncoding.EncodeToString(userJSON)
+		userJSONBase64 := base64.URLEncoding.EncodeToString(userJSON)
```

Comment:
is this an unrelated bugfix?

Reply:
That other PR landed the fix on v2 so we can get the benefit immediately. I had to wait for it to land, pull those changes into the feat/inline-contexts branch, and then update this PR accordingly.

tl;dr -- removed!